### DISCUSSION TITLE
[Bug] How can an external machine access 8555 RPC port

### DISCUSSION THREAD

**Discussion by PrernaPadhee** [Chia-Network/chia-blockchain#10378] [Support]
### What happened?

My use-case is such that I need other machines to access the chia node. Using the node via localhost is not a viable solution for me. Kindly help with the same.

### Version

chia:1.2.11

### What platform are you using?

Linux

### What ui mode are you using?

CLI

**Comment by scrutinously** [comment]
I believe you would need to have your keys on the external machine, then call those as the key and cert arguments. Obviously also replace the localhost address with the IP of the machine running chia.

**Reply by PrernaPadhee** [reply]
should i be changing the IP in config.yaml ?
